ethstaker_deposit/utils/validation.py

```diff
@@ -138,6 +139,14 @@ def validate_deposit(deposit_data_dict: Dict[str, Any], credential: Credential =
 def validate_password_strength(password: str) -> str:
     if len(password) < 8:
         raise ValidationError(load_text(['msg_password_length']))
+
+    encoding = sys.stdin.encoding.lower()
+    if encoding != 'utf-8' and not password.isascii():
+        if sys.platform == 'win32':
+            raise ValidationError(load_text(['msg_password_utf8']))
+        else:
+            raise ValidationError(load_text(['msg_password_utf8', 'msg_password_utf8_switch']))
```

Comment:
With my example unit test, I was having an issue with this where I _think_ it was looking for `validate_password_strength.msg_password_utf8.msg_password_utf8_switch` instead of showing both.

Reply:
Good catch. Fixed.